firmware/arduino/chessboard/chessboard.ino

```diff
@@ -78,6 +72,15 @@ int currentX, currentY;
 // Number of different slope settings for each quarter circle
 #define NUM_SLOPES_PER_QUARTER_CIRCLE 9
 
+// Movement function status codes
+enum MovementStatus
+{
+  SUCCESS = '0',
+  HIT_X_ENDSTOP = '1',
+  HIT_Y_ENDSTOP = '2',
+  INVALID_ARGS = '3'
```

Comment:
These should be ints instead of chars, I know that's different from Chase's. sorry for not making that clear.

Reply:
```c
enum MovementStatus
{
   SUCCESS = 0,
   HIT_X_ENDSTOP = 1,
   HIT_Y_ENDSTOP = 2,
   INVALID_ARGS = 3
}
```

Reply:
Also, since an enum can be declared anywhere (unlike preprocessors), this should be in motorDriver.ino